packages/block-library/src/image/edit.native.js

```diff
@@ -39,6 +40,44 @@ export default function ImageEdit( { attributes } ) {
 				resizeMethod="scale"
 				source={ { uri: url } }
 			/>
+			{ CaptionTextInput( setAttributes, caption, isSelected ) }
 		</View>
 	);
 }
+
+const CaptionTextInput = ( setAttributes, caption, isSelected ) => {
+	const captionString = [] + caption; // caption can be a string or an array.
+
+	if ( ! captionString.length && ! isSelected ) {
+		return null;
+	}
+
+	const textContainer = ( children ) => {
```

Comment:
Any particular reason we fall back to plain textview instead of just letting the editing component in?